src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -226,7 +236,7 @@ internal static void AddFunctionError(Dictionary<string, Collection<string>> fun
             await CallAsync(methodInfo, arguments, cancellationToken);
         }
 
-        protected virtual void Initialize()
+        protected virtual void Initialize(ScriptSettingsManager settingsManager)
```

Comment:
The instance already has a settings manager passed to it on construction, so we shouldn't need to modify this signature. Just use _settingsManager